docs/prow/kind-jobs.md

```diff
@@ -0,0 +1,74 @@
+# kind Jobs
+
+This document describes how to execute kind jobs locally without Prow cluster.
```

Comment:
```suggestion
This document describes how to trigger kind jobs locally without a Prow cluster.
```

Reply:
Hm... I'm not sure if the trigger is a good word - for me, it means something asynchronous. This instruction explains how to run a job on Docker, not Kubernetes, locally and in this scenario it is synchronous.

Reply:
ok, so maybe `run` would be the most appropriate?
```suggestion
This document describes how to run kind jobs locally without a Prow cluster.
```